tools/util/profiles/cqlstress_blob_profile.yaml

```diff
@@ -0,0 +1,19 @@
+table: test_table
+
+table_definition:
+  CREATE TABLE IF NOT EXISTS test_table (
+    pkey blob PRIMARY KEY,
+    c1 blob,
+    c2 blob,
+    c3 blob
+   );
+
+keyspace: cqlstress_blob_keyspace
+
+keyspace_definition: |
+  CREATE KEYSPACE IF NOT EXISTS cqlstress_blob_keyspace 
+  WITH replication = { 'class': 'SimpleStrategy', 'replication_factor': 1};
```

Comment:
...and I can see `|` here. Why is it so?

Reply:
It's because of the colons (`:`) contained in the query itself (e.g. `'class':`). This way, they are escaped and not interpreted as another parameter definition. If you remove the pipe symbol, the yaml parser will fail to parse the file. The pipe symbol is not needed for `table_definition`, but I might add it just for the consistency. WDYT?

Reply:
Yes, please. If it makes sense to always escape CQL statements, then let's do that consistently.